packages/tailwindcss/package.json

```diff
@@ -27,14 +27,26 @@
       "require": "./src/compat/colors.cts",
       "import": "./src/compat/colors.ts"
     },
+    "./colors.js": {
+      "require": "./src/compat/colors.cts",
+      "import": "./src/compat/colors.ts"
+    },
     "./defaultTheme": {
       "require": "./src/compat/default-theme.cts",
       "import": "./src/compat/default-theme.ts"
     },
+    "./defaultTheme.js": {
+      "require": "./src/compat/default-theme.cts",
+      "import": "./src/compat/default-theme.ts"
+    },
     "./plugin": {
       "require": "./src/plugin.cts",
       "import": "./src/plugin.ts"
     },
+    "./plugin.js": {
+      "require": "./src/plugin.cts",
+      "import": "./src/plugin.ts"
+    },
```

Comment:
These need to be updated in the publishConfig too

Reply:
ah of course 🫠